### PR TITLE
explorer: compose Transactions page

### LINKED ISSUE #1721: explorer: Compose `Transactions` page
Compose the Blocks page that is comprised of the Transactions card used on the homepage.

Reference Figma [file](https://www.figma.com/file/INYlGpXEc8tIHCEPNlPyfd/Explorer-2.0?type=design&node-id=747-13223&mode=dev)

Image

**Unit tests**
- It should check if data is being displayed in the table
- It should check if the loading state is displayed after triggering the load on button click
- It should display the correct message when there is no more data to load

### PR DESCRIPTION
**Pull request by deuch13**
Resolves #1721

### CODE REVIEW

File: explorer/src/routes/blocks/+page.svelte [hunk 1]
```diff
@@ -1,3 +1,28 @@
...
+  import { createPollingDataStore } from "$lib/dusk/svelte-stores";
+  import { onNetworkChange } from "$lib/lifecyles";
+
+  const pollingDataStore = createPollingDataStore(
+    duskAPI.getBlocks,
+    $appStore.fetchInterval
+  );
+
+  onNetworkChange((network) => {
+    pollingDataStore.stop();
```
Comment (ascartabelli) [inline explorer/src/routes/blocks/+page.svelte:14]: Remember to add a `stop` call in the `onDestroy` hook.

File: explorer/src/routes/transactions/+page.svelte [hunk 2]
```diff
@@ -1,3 +1,28 @@
...
+  import { createPollingDataStore } from "$lib/dusk/svelte-stores";
+  import { onNetworkChange } from "$lib/lifecyles";
+
+  const pollingDataStore = createPollingDataStore(
+    duskAPI.getTransactions,
+    $appStore.fetchInterval
+  );
+
+  onNetworkChange((network) => {
+    pollingDataStore.stop();
```
Comment (ascartabelli) [inline explorer/src/routes/transactions/+page.svelte:14]: Remember to add a `stop` call in the `onDestroy` hook.

**Review by ascartabelli — CHANGES_REQUESTED** [review]
Same minor comments I made in the Blocks PR: only important thing is to stop the polling in the `onDestroy` hook.

**Review by ascartabelli — CHANGES_REQUESTED** [review]
Just remove the test, and open an issue for later.

Good to merge otherwise.

File: explorer/src/lib/components/__tests__/TransactionsCard.spec.js [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+
+    const button = getByRole("button");
+
+    const showMoreIncrement = 15;
+
+    const clicks = Math.ceil(data.length / showMoreIncrement) - 1;
+
+    Array.from({ length: clicks }).forEach(async () => {
+      await fireEvent.click(button);
+    });
```
[the remark this answers is not in the document]
Comment (deuch13) [reply explorer/src/lib/components/__tests__/TransactionsCard.spec.js]: In this case I would remove the data slice and leave the full data mock.

**Review by ascartabelli — APPROVED** [review]

**Pull request merged**